spec/index.adoc

```diff
@@ -141,8 +141,13 @@ NOTE: Derived IDs are currently not explicitly used in this document, but they m
 +
 This has the form `0x04`, and is used for the anonymous caller. It can be used in call and query requests without a signature.
 
-When the IC creates a _fresh_ id, it never creates a self-authenticating id, an anonymous id or an id derived from what could be a canister or user.
+5. _Reserved ids_
++
+These have the form of `blob · 0xff`, with no restriction on the blob length.
```

Comment:
"no restriction": there is still the global restriction that any principal id is at most 29 bytes, right? That means that the blob is at most 28 bytes long.